openduck-py/openduck_py/routers/voice.py

```diff
@@ -242,6 +251,34 @@ def _check_for_exceptions(response_task: Optional[asyncio.Task]) -> bool:
         return reset_state
 
 
+async def log_event(db: AsyncSession, session_id: str, event: EventName, meta: Optional[Dict[str, str]] = None, audio: Optional[np.ndarray] = None):
+    if audio is not None:
+        counter = 0
+        path = f"/openduck-py/openduck-py/logs/{session_id}/{event}_{counter}.wav"
```

Comment:
Can we do something like this so it works in docker and not in docker? `pathlib.Path(__file__)` I can't remember what the exact syntax is